buildpacktoml_test.go

```diff
@@ -767,6 +869,78 @@ func testBuildpackTOML(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 		})
+
+		when("buildpack api < 0.6", func() {
+			it.Before(func() {
+				bpTOML.API = "0.5"
+				mockEnv.EXPECT().WithPlatform(platformDir).Return(append(os.Environ(), "TEST_ENV=Av1"), nil)
+			})
+
+			when("platform API < 0.6", func() {
```

Comment:
https://github.com/buildpacks/lifecycle/pull/520/files#r568171284 would help avoid the combinatorial explosion of buildpack & platform apis.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/520/commits/845f01c70137cd8ddf8c399a1b6a93f3199d6b72.